sdcm/tester.py

```diff
@@ -1076,17 +1078,35 @@ def run_stress_cassandra_thread(self, stress_cmd, duration=None, stress_num=1, k
         return cs_thread
 
     def run_stress_thread_bench(self, stress_cmd, duration=None, stress_num=1, keyspace_num=1, profile=None, prefix='',  # pylint: disable=too-many-arguments,unused-argument
-                                round_robin=False, stats_aggregate_cmds=True, keyspace_name=None, use_single_loader=False):  # pylint: disable=too-many-arguments,unused-argument
+                                round_robin=False, stats_aggregate_cmds=True, keyspace_name=None, use_single_loader=False,
+                                stop_test_on_failure=True):  # pylint: disable=too-many-arguments,unused-argument
 
         timeout = self.get_duration(duration)
+        stop_test_on_failure = False if not self.params.get("stop_test_on_stress_failure") else stop_test_on_failure
+
+        if self.params.get("authenticator") and self.params.get("authenticator") == "PasswordAuthenticator":
+            authenticator_user = self.params.get("authenticator_user")
+            authenticator_password = self.params.get("authenticator_password")
+
+            if authenticator_user and authenticator_password:
+                stress_cmd += f" -username {authenticator_user} -password {authenticator_password}"
+            else:
+                event_type = 'failure'
```

Comment:
I think this validation should be done on SCTConfiguration level, not in runtime

Reply:
changed

Reply:
I found that in cluster.py/BaseCluster._create_session allow  that usename and password may be None:
```
        authenticator = self.params.get('authenticator')
        if user is None and password is None and (authenticator and authenticator == 'PasswordAuthenticator'):
            user = self.params.get('authenticator_user', default='cassandra')
            password = self.params.get('authenticator_password', default='cassandra')

```

@bentsi  @dkropachev 
It looks like the problem because of it's inconsistent with cassandra-stress usage. 
I suggest to remove this